docs/patterns/category/design-implementation.md

```diff
@@ -20,6 +20,13 @@ categories: featured
 
 Good design encompasses factors such as consistency and coherence in component design and deployment, maintainability to simplify administration and development, and reusability to allow components and subsystems to be used in other applications and in other scenarios. Decisions made during the design and implementation phase have a huge impact on the quality and the total cost of ownership of cloud hosted applications and services.
 
+> [!NOTE]  
+> In distributed systems:
+> - Versioning isn't simple.
+> - Observabality is not optional but mandatory.
+> 
+
+
```

Comment:
Convert this text to be inline instead of using the NOTE syntax.  Find a way to stress the importance of these topics and relate it to the list below (presumably these will help solve those problems)